meson.build

```diff
@@ -46,14 +46,14 @@ configure_file(configuration : conf_data,
               )
 
 hardware_isolation_sources = [
-        'hardware_isolation_entry.cpp',
-        'hardware_isolation_manager.cpp',
-        'hardware_isolation_manager_main.cpp',
-        'isolatable_hardwares.cpp',
-        'openpower_guard_interface.cpp',
-        'phal_devtree_utils.cpp',
-        'utils.cpp',
-        'watch.cpp'
+        'src/hardware_isolation_main.cpp',
```

Comment:
you can add "incdir = include_directories("xxx")"  . this will help to serch the header files. instead of specifying complete path in in #include.

Reply:
Agree but, I am thinking below two points
- We cannot have the same file name in the different directory
For example,
```
include/common/common_types.hpp
include/hw_isolation_record/common_types.hpp

meson.build

inc_dir_list = include_directories('include/common', `include/hw_isolation_record`)

foo.cpp

#include "common_types.hpp" - the compiler will pick based on the first identified file (aka search path preference) but, we will lose the expected header file.
```
- Developer or User don't know which header file need to look at if have the same file name in a different directory

Reply:
> Agree but, I am thinking below two points
> 
>     * We cannot have the same file name in the different directory
>       For example,
> 
> 
> ```
> include/common/common_types.hpp
> include/hw_isolation_record/common_types.hpp
> 
> meson.build
> 
> inc_dir_list = include_directories('include/common', `include/hw_isolation_record`)
> 
> foo.cpp
> 
> #include "common_types.hpp" - the compiler will pick based on the first identified file (aka search path preference) but, we will lose the expected header file.
> ```
> 
>     * Developer or User don't know which header file need to look at if have the same file name in a different directory
This will popup in build time right?

Reply:
Do you got real use for using same file name here?

Incase you want to sue same file name , i would remove the serch for the file path.

Reply:
> This will popup in build time right?

Yes, the compiler will decide that during compilation and throw the error if expected things are not present in the identified header file.

> Do you got real use for using same file name here?

Not now,

> Incase you want to sue same file name , i would remove the serch for the file path.

in the meson.build? include_directories and explicit inclusion of the path? If so, In this patch we are doing the same.

If some different way, please share (it will be new learning for me)

Reply:
LGTM ,